frontend/src/components/ViewTrips/trips-container.js

```diff
@@ -63,7 +18,13 @@ function serveTrips(querySnapshot, handleEditTrip, refreshTripsContainer) {
 function getErrorElement(error) {
   return new Promise(function(resolve) {
     console.log(`Error in Trips Container: ${error}`);
-  resolve(( <div><p>Error: Unable to load your trips.</p></div> ));
+    resolve(
+      <div>
+        <p>Oops, it looks like we were unable to load your trips.
+                      Please wait a few minutes and try again.
+        </p>
+      </div>
```

Comment:
One of my PRs created an `Errors` directory, so once that's merged do you think you could turn this function into a component and place it there? An issue could be created in the meantime.

Reply:
Sure thing. I will add an issue right now.